src/schema/associations.ts

```diff
@@ -123,33 +146,37 @@ export async function buildAssociations(
         rule.target.suffix,
         rule.target?.entities ?? [],
       ).next().value
```

Comment:
The EMG examples do not have this case, but with multiple `coordsystem.json`s, you could imagine having some at the root and some at the leaf.

We currently strongly assume (and implement with `.next().value`) that the first level found is the level of the association, which has worked for single-file associations. This probably needs to be relaxed, but the logic will be more intricate. I think you would need to take the first `coordsystem.json` with a given `space` entity, not the full collection.

For now this could be noted as a limitation in the validator implementation.